stdlib/sys.pyi

```diff
@@ -219,22 +344,24 @@ if sys.version_info < (3, 9):
     def setcheckinterval(__n: int) -> None: ...  # deprecated
 
 if sys.version_info >= (3, 8):
-    # not exported by sys
-    class UnraisableHookArgs:
+    class _UnraisableHookArgs:
```

Comment:
I think we shouldn't break pytest just because we can, and the comment seems to suggest that exposing `UnraisableHookArgs` is intentional. I'm open to other opinions though.

Reply:
The comment does seem to suggest that it's intentional, but I don't understand why, and the PR that added it (https://github.com/python/typeshed/pull/3358) doesn't shed any light.

Reply:
pytest annotates a few different thing with this. Copying the definition to pytest wouldn't work, so pytest would have to use `sys._UnraisableHookArgs` or `Any`. Neither of those feels like an improvement to me.

Reply:
Okay, I'll revert the change.